src/codegen/sys/lib_.rs

```diff
@@ -158,7 +153,7 @@ fn generate_constants(w: &mut Write, env: &Env, constants: &[Constant]) -> Resul
             comment = "//";
         }
         let mut value = constant.value.clone();
-        if type_ == "&str" {
+        if type_ == "*mut c_char" {
             type_ = "&'static str".into();
             value = format!("r##\"{}\"##", value);
         } else if type_ == "Glyph" && env.config.library_name == "Pango"  {
```

Comment:
This was probably also the reason for the Pango special case here. Does someone remember/know?